_docs/ops/managing-users.md

```diff
@@ -56,52 +56,52 @@ Make sure you have a copy of the [cg-scripts repository](https://github.com/18F/
 
 ### Creating Admins
 
-First, target and get a token for the main CloudFoundry UAA, and make the user a CloudFoundry admin using their GSA email address.
+Make the user a Production CloudFoundry admin using their GSA email address.
 
 ```sh
-cd /path/to/cg-scripts
-# on a jumpbox, run this instead of the next two commands: ./uaa/login.sh
-uaac target <CF_UAA_FQDN>
-uaac token client get admin -s <CF_UAA_ADMINCLIENT_PASSPHRASE>
+# on a production jumpbox, run:
+cd ./cg-scripts
+./uaa/login.sh
 ./make-cf-admin.sh <EMAIL_ADDRESS>
+# For global auditor, instead use 
+# ./add-global-cf-auditor-permissions.sh <EMAIL_ADDRESS>`
```

Comment:
Do we need a ` at the beginning of this line, or should we remove the trailing one?

Reply:
Thanks for catching that. Fixed.